include/circt/Dialect/FIRRTL/FIRRTLStructure.td

```diff
@@ -141,9 +142,8 @@ def NonLocalAnchor : FIRRTLOp<"nla",
     annotations to anchor to in the global scope.  This lets components of the
     path point to a common entity.
   }];
-  let arguments = (ins SymbolNameAttr:$sym_name,
-                       FlatSymbolRefArrayAttr:$modpath, StrArrayAttr:$namepath);
+  let arguments = (ins SymbolNameAttr:$sym_name, NameRefArrayAttr:$namepath);
   let results = (outs);
-  let assemblyFormat = [{ $sym_name $modpath $namepath attr-dict}];
+  let assemblyFormat = [{ $sym_name $namepath attr-dict}];
```

Comment:
I love how this replaces path elements with just one entity that we have an easier way to reason about!